### DISCUSSION TITLE
Use multiple temperature sensors, calibration offsets

### DISCUSSION THREAD

**Discussion by teeten** [jmcollin78/versatile_thermostat#336] [Ideas]
I am using the thermostat in switch mode with electric heaters and simple Zigbee relays.
I love that it can manage multiple heaters.

It would be nice if it would also use multiple temperature sensors.

One mode could be to use the latest temperature reported by one of the sensors. This would act as a high availability feature, so if one sensor breaks down or runs out of battery, the thermostat would naturally use the other one. It would also reduce the probability of going into safety mode.

One option could be to use the average temperature from multiple sensors, only averaging temperatures reported in the last 10 minutes (for example).

One other option would be to allow offsetting temperature values for each sensor to facilitate calibration. This is something we should have in the sensor itself, but it would make it super easy if the thermostat could handle it.

**Comment by jmcollin78** [comment]
Hello,

> It would be nice if it would also use multiple temperature sensors.

this have been requested many times and answered as well. Please search into the issue / discussions.

**Reply by jmcollin78** [reply]
Finally I don't find the response in Github. I think it is on the forum. 

This one gives you an answer :
#118
